### ISSUE TITLE
multiple address components of the same type cause error

### ISSUE THREAD

**Issue by qing-duan** [degauss-org/postal#11]
I encountered an error when running postal but not other containers. Below is the error. What could be wrong?
*! Column names `city`, `state`, and `postcode` must not be duplicated.
Use .name_repair to specify repair.
Caused by error in `repaired_names()`:
! Names must be unique.
âœ– These names are duplicated:
  \* "city" at locations 2 and 5.
  \* "state" at locations 3 and 6.
  \* "postcode" at locations 4 and 7.*

**Comment by cole-brokamp** [comment]
Thanks, Qing!  Can you please create a [minimal reproducible example (MRE)](https://stackoverflow.com/help/minimal-reproducible-example) of the error, bug, or issue that you are encountering.  Specifically, is there an example input file and DeGAUSS command that generates this specific error?

> An MRE is intended to reproduce an error using the smallest amount of code, which will save troubleshooting time and effort. It also ensures that the problem is related to code in this repository, as opposed to other software tools being used alongside DeGAUSS or to your specific input data.  Once we can reproduce the specific problem, then we can work to troubleshoot it.

> DeGAUSS is often used with sensitive and private data that cannot be shared, so using a contrived, random, or simulated example dataset might be necessary.  Every DeGAUSS repository has [test](https://github.com/degauss-org/geocoder/blob/master/test/my_address_file.csv) [input](https://github.com/degauss-org/census_block_group/blob/master/test/my_address_file_geocoded.csv) files that are useful for troubleshooting and trying to create a MRE.

**Comment by qing-duan** [comment]
Thank you for the quick response Cole! 
Here is the example input file where I encountered the issue:
[example_addresses.csv](https://github.com/degauss-org/postal/files/9832331/example_addresses.csv)
The DeGAUSS command is:
docker run --rm -v $PWD:/tmp ghcr.io/degauss-org/postal:0.1.2 example_addresses.csv

I ran it in r using degauss_run().

The full error messages is:
â”€â”€ Welcome to DeGAUSS! â”€â”€

â€¢ You are using postal, version 0.1.2
â€¢ This container returns normalized and parsed addresses
â€¢ <https://degauss.org/postal>

âœ” imported data from degauss_5ba022bf4a9d.csv
â„¹ removing non-alphanumeric characters...
â„¹ removing excess whitespace...
â„¹ parsing addresses...
Error:
! Column names `city`, `state`, and `postcode` must not be duplicated.
Use .name_repair to specify repair.
Caused by error in `repaired_names()`:
! Names must be unique.
âœ– These names are duplicated:
 \* "city" at locations 2 and 5.
  \* "state" at locations 3 and 6.
  \* "postcode" at locations 4 and 7.
Backtrace:
     â–†
  1. â”œâ”€dplyr::rename_with(...)
  2. â”œâ”€dplyr::bind_rows(...)
  3. â”‚ â””â”€rlang::list2(...)
  4. â”œâ”€purrr::modify(...)
  5. â””â”€purrr:::modify.default(...)
  6.   â”œâ”€tibble (local) .f(.x[[i]], ...)
  7.   â””â”€tibble:::as_tibble.list(.x[[i]], ...)
  \8.     â””â”€tibble:::lst_to_tibble(x, .rows, .name_repair, col_lengths(x))
 \9.       â””â”€tibble:::set_repaired_names(x, repair_hint = TRUE, .name_repair)
 \10.         â”œâ”€rlang::set_names(...)
\ 11.         â””â”€tibble:::repaired_names(= NULL)
 \12.           â”œâ”€tibble:::subclass_name_repair_errors(...)
\13.           â”‚ â””â”€base::withCallingHandlers(...)
 \14.           â””â”€vctrs::vec_as_names(...)
 \15.             â””â”€vctrs (local) `<fn>`()
 \16.               â””â”€vctrs:::validate_unique(names = names, arg = arg, call = call)
 \17.                 â””â”€vctrs:::stop_names_must_be_unique(names, arg, call = call)
 \18.                   â””â”€vctrs:::stop_names(...)
 \19.                     â””â”€vctrs:::stop_vctrs(...)
 \20.                       â””â”€rlang::abort(message, class = c(class, "vctrs_error"), ..., call = vctrs_error_call(call))
Execution halted
[1] 1

**Comment by cole-brokamp** [comment]
Perfect -- that's very helpful!  I ran the command with your example data and was able to reproduce the same error you got, so I definitely think this is a bug.  

Looking over the example address file, I found a suspicious looking one that had the cit and state in there twice.  After removing this one, it worked successfully.  

I think the fix here will be for me to pick the *first* of each component if more than one of them is found.
